src/api/SFA.DAS.Provider.Events.Api.Types/PaymentStatistics.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace SFA.DAS.Provider.Events.Api.Types
+{
+    public class PaymentStatistics
+    {
+        public int TotalPayments { get; set; }
```

Comment:
Can this be renamed to TotalNumberOfPayments (assuming that's what it is)?

Reply:
done